src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/UpdateSentenceDateService.kt

```diff
@@ -72,6 +72,9 @@ class UpdateSentenceDateService(
         recordAuditEvent(updatedLicenceEntity, "Sentence dates updated")
         recordAuditEvent(timedOutLicence, "Licence automatically timed out after sentence dates update")
         recordLicenceEvent(timedOutLicence, "Licence automatically timed out after sentence dates update")
```

Comment:
wonder if we had a:
```kt
licenceService.timeout(licence, reason = ""Licence automatically timed out after sentence dates update")
```
that we could use in both places - which did the `timeout` the persisting, the auditing and the notifying?